plugins/katello/3.15/release_notes/release_notes.md

```diff
@@ -0,0 +1,78 @@
+---
+title: Release Notes
+---
+
+# Katello {{ page.version }} Release Notes
+
+For the full release notes, see the [Changelog](https://github.com/Katello/katello/blob/master/CHANGELOG.md).
+
+## Features
+
+### Subscriptions
```

Comment:
We need to get back to having actual release notes instread of an abbreviated changelog. It's my fault I set that example last release. Maybe I intended to use what I had there as a baseline for the real notes? Who knows...

I took a shot for 3.15:

**Features**

- Improved experience for Simple Content Access users with clarified messaging and UI tweaks
- Errata can be searched by their issued date
- `hammer gpg` has been renamed to `hammer content-credentials` to match the UI
- Host updates will no longer use Dynflow for increased performance
- << Something about Pulp3? >>
- Improved performance of Subscriptions list API
- Content count information is now stored on Content View Versions directly for increased performance

**Deprecations**

- Puppet and OSTree repository types will no longer be available in Katello 3.16

@jlsherrill @jjeffers you guys might be able to articulate a pulp3 related bullet point(s) in terms a user would care about

Reply:
We may also want to mention that katello-agent is deprecated here, even though we're officially deprecating in 3.16 - I don't think it can hurt. cross-linking https://github.com/Katello/katello/pull/8588